src/server/pegasus_mutation_duplicator.cpp

```diff
@@ -234,7 +242,8 @@ void pegasus_mutation_duplicator::duplicate(mutation_tuple_set muts, callback cb
             batch_bytes += raw_message.length();
         }
 
-        if (batch_count == muts.size() || batch_bytes >= FLAGS_duplicate_log_batch_bytes) {
+        if (batch_count == muts.size() || batch_bytes >= FLAGS_duplicate_log_batch_bytes ||
+            batch_bytes >= dsn::replication::FLAGS_dup_max_allowed_write_size) {
```

Comment:
Once `batch_bytes > dsn::replication::FLAGS_dup_max_allowed_write_size`, would the message be rejected by   backup(follower) cluster ?

Reply:
Actually `dup_max_allowed_write_size` of master cluster relates to `max_allowed_write_size` of follower cluster. Once requests are rejected by follower cluster, `max_allowed_write_size` of follower cluster needs to be increased.